cmd/cli/app/get.go

```diff
@@ -75,21 +76,36 @@ var getCmd = &cobra.Command{
 
 			resp, err := rekorClient.Entries.GetLogEntryByUUID(params)
 			if err != nil {
-				log.Fatal(err)
+				return nil, err
 			}
 
-			for _, entry := range resp.Payload {
-				bytes, err := entry.MarshalBinary()
-				if err != nil {
-					log.Fatal(err)
+			for k, entry := range resp.Payload {
+				if k != uuid {
+					continue
 				}
-				log.Info(string(bytes))
+				return parseEntry(entry)
 			}
-			return
 		}
 
-		log.Fatal("either --uuid or --log-index must be specified")
-	},
+		return nil, errors.New("either --uuid or --log-index must be specified")
+	}),
+}
+
+func parseEntry(e models.LogEntryAnon) (interface{}, error) {
```

Comment:
These feels fragile to me but barring a cast of `e.Body` to string and doing the base64 decode directly, I can't think of a better way to accomplish this.

Reply:
Yeah I tried a bunch of ways and got confused because the base64 decoding was happening automatically sometimes (I think []byte serialization takes care of that?), but not others.